### DISCUSSION TITLE
Keycloak realm deployment issue in ubuntu 20

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#565] [Bugs] [answered]
Hi Team!

we are facing an issue in keycloak realm deployment. Below are the error logs.
Kindly suggest.
@santhosh-tg 

#####################
STDERR:

zip_safe flag not set; analyzing archive contents...
zip_safe flag not set; analyzing archive contents...
/usr/lib/python2.7/dist-packages/setuptools/dist.py:476: UserWarning: Normalizing '2023.05.07' to '2023.5.7'
  normalized_version,
warning: no previously-included files found matching '.github/'
warning: manifest_maker: MANIFEST.in, line 4: 'recursive-exclude' expects <dir> <pattern1> <pattern2> ...

  File "build/bdist.linux-x86_64/egg/certifi/core.py", line 17
    def where() -> str:
                ^
SyntaxError: invalid syntax

  File "/usr/local/lib/python2.7/dist-packages/certifi-2023.5.7-py2.7.egg/certifi/core.py", line 17
    def where() -> str:
                ^
SyntaxError: invalid syntax

src/_fastmath.c:31:10: fatal error: Python.h: No such file or directory
   31 | #include "Python.h"
      |          ^~~~~~~~~~
compilation terminated.
error: Setup script exited with error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
###########

**Comment by chitranshu-keshav** [comment]
@santhosh-tg @gandham-santhosh Can you please get back on this

**Reply by santhosh-tg** [reply]
The issue seems to be with **certifi-2023.5.7** which requires python3 but from the error msg posted you are using python-2.7.

Please install a  version supported by 2.7 and  give a try.

**Reply by gohilamariappan** [reply, marked as answer]
@santhosh-tg Thank you.   Issue resolved. below one is the solution.
pip2 install certifi==2018.1.18.
